### DISCUSSION TITLE
module 'metadata.generated.schema.type.basic' has no attribute 'Markdown'

### DISCUSSION THREAD

**Discussion by stoveZhu** [open-metadata/OpenMetadata#16448] [Q&A] [answered]
I write a great expectation checkpoint and want to send it to my openmetadata platform. The checkpoint runs well on Win but shows error: `module 'metadata.generated.schema.type.basic' has no attribute 'Markdown'` when the code runs on Ubuntu 24.04 virtual machine. 
my checkpoint code:
`name: my_checkpoint
config_version: 1.0
template_name:
module_name: great_expectations.checkpoint
class_name: Checkpoint
run_name_template:
expectation_suite_name:
batch_request: {}
action_list:
  - name: my_action_name
    action:
      module_name: metadata.great_expectations.action
      class_name: OpenMetadataValidationAction
      config_file_path: /home/stove/airflow/dags/include/gx
      database_service_name: basic_his
  - name: store_validation_result
    action:
      class_name: StoreValidationResultAction
  - name: store_evaluation_params
    action:
      class_name: StoreEvaluationParametersAction
  - name: update_data_docs
    action:
      class_name: UpdateDataDocsAction
evaluation_parameters: {}
runtime_configuration: {}
validations:
  - batch_request:
      datasource_name: my_datasource
      data_asset_name: A
    expectation_suite_name: my_suite
profilers: []
ge_cloud_id:
expectation_suite_ge_cloud_id:
`

python version: 3.12.3
package version:
great_expectation   0.18.14
openmetadata-ingestion  1.3.2.0

**Comment by Prajwal214** [comment, marked as answer]
@stoveZhu Great Expectations has released version 0.18.14, which breaks one of the interfaces we use. Please pin the version to 0.18.13 to ensure it works.

https://openmetadata.slack.com/archives/C02B6955S4S/p1716444768421819
